source/c.txt

```diff
@@ -75,6 +76,35 @@ Connect to MongoDB Atlas
 See `Advanced Connections <http://mongoc.org/libmongoc/current/advanced-connections.html>`__
 for more ways to connect.
 
+Versioned API
+-------------
+
+You can use the `Versioned API
+<https://docs.mongodb.com/v5.0/reference/versioned-api/>`__ feature
+starting with MongoDB Server version 5.0 and C Driver version 1.18. When
+you use the Versioned API feature, you can update your driver or server
+without worrying about backward compatibility issues for any commands
+covered by the Versioned API. To use this feature, specify the version
+of the Versioned API:
```

Comment:
How would you feel about using the text "construct a client, specifying a version of the Versioned API" as that more accurately describes what the code shows and what's required?

Reply:
yeah I think the wording in this sentence needs some work. How is this:
"To use this feature, construct a MongoDB client instance with the version of the Versioned API."

Reply:
Personal opinion: "Z with Y" often omits information on how the two relate. In this case, here is a reasonable way in which someone -- especially new to programming or these concepts -- could read the proposed sentence:

Instruct the version of the Versioned API to build a client instance so that you can use the feature.

This reading isn't correct since the version is a number, not some factory class that constructs new instances of a client class.

Reply:
Okay, I will use the "..., specifying ..." structure. thanks!

Reply:
(of course feel free to come up with your own solutions)